core/src/main/java/org/libreoffice/lots/document/DocumentLoader.java

```diff
@@ -60,9 +60,9 @@ public class DocumentLoader
   private LoadingCache<String, ByteBuffer> cache;
 
   /**
-   * Zugriff auf den DocumentLoader als Singleton.
+   * Access to the DocumentLoader as a singleton.
    *
-   * @return Singleton-Instanz des DocumentLoaders
+   * @return Singleton-Instance of Document_Loaders
```

Comment:
Still not correct. Should say "Singleton instance of DocumentLoader".

Reply:
Ok! Let me correct it.